src/Akka.HealthCheck.IHealthCheckExample/Program.cs

```diff
@@ -0,0 +1,54 @@
+using Akka.Cluster.Hosting;
+using Akka.HealthCheck.Cluster;
+using Akka.HealthCheck.IHealthCheckExample;
+using Akka.HealthCheck.IHealthCheckExample.HealthChecks;
+using Akka.HealthCheck.Liveness;
+using Akka.HealthCheck.Persistence;
+using Akka.HealthCheck.Readiness;
+using Akka.Hosting;
+using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+builder.Services.AddAkka("actor-system", (configurationBuilder, serviceProvider) =>
+{
+    configurationBuilder
+    .WithClustering()
+    .WithActors((actorSystem, registry) =>
+    {
+        // We need to start the Probe actors which we need in our HealthChecks
+        actorSystem.CreateAndRegisterHealthProbe<DefaultLivenessProbe>(registry, new DefaultLivenessProvider(actorSystem));
```

Comment:
Maybe think of a way to automatically register these in the ActorRegistry based on some config (make an Akka.Hosting helper for this?)